### ISSUE TITLE
Green/yellow/red colors on the minigame order overlay instead of green/white/white

### ISSUE THREAD

**Issue by Speeddymon** [hex-agon/mastering-mixology#57]
Hi, on the minigame's overlay where this plugin colors the highest XP mixture if you've chosen Favor XP strategy, please add additional highlighting.

For example if I have:

(H) LLL
(H) LLL
(Cr) LLL

The first (H) LLL is green but the other two mixtures are white. I think the second (H) LLL should be green too and (Cr) LLL should be red.

And if I have:

(H) AAA
(H) MMA
(Co) AAA

The (H) MMA is green but the other two mixtures are white. Which of the AAA mixtures is higher XP? If they are the same, both should be red; if not, then one could be yellow and one could be red.

Finally if I have:

(Cr) AAA
(Co) LLL
(Cr) MMM

The (Co) LLL is green. The others could be colored the same as I described my second example above.

In summary:
I want to know if there's a tie for the either the highest or the lowest XP and I want to know which is the second highest or third highest if there's no tie, because I'm prioritizing a certain paste at the moment but want to keep the plugin on Favor XP strategy, so I might not always make the thing that's highest XP if it's not using a paste I'm prioritizing, but I don't necessarily want to make the lowest XP mixture either.

Thanks for your consideration.

**Comment by raiyni** [comment]
Strategy has been removed
